Makefile

```diff
@@ -1,13 +1,23 @@
 GOPATH ?= $(shell go env GOPATH)
 
+.PHONY: build
+build:
+	mkdir -p bin
+	GOBIN="$(CURDIR)/bin/" go install ./cmd/...
+
 .PHONY: lint
 lint:
 	bash scripts/install-lint.sh
 	${GOPATH}/bin/golangci-lint run
 
+.PHONY: format
+format:
+	go fmt ./...
+
+.PHONY: test
 test:
-	go test ./...
+	go test -count=1 ./...
```

Comment:
Made the test results uncached to increase our chance of catching tests that sometimes succeed and sometimes fail.